openslides/core/static/templates/index.html

```diff
@@ -32,35 +32,91 @@
-                                password: <input type="password" ng-model="password"><br>
-                                <input type="submit" ng-click="login(username, password)" value="Save" />
-                            </form>
+                        <div class="modal" id="loginFormModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+                            <div class="modal-dialog modal-sm">
+                                <div class="modal-content">
+                                    <div class="modal-header">
+                                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                                            <span aria-hidden="true">&times;</span>
+                                        </button>
+                                        <h4 class="modal-title" id="myModalLabel" translate>Login</h4>
+                                    </div>
+                                    <div class="modal-body">
+                                        <form>
+                                          <div class="form-group">
+                                            <label for="inputUsername" translate>Username</label>
+                                            <input type="text" ng-model="username" class="form-control input-lg" id="inputUsername">
+                                          </div>
+                                          <div class="form-group">
+                                            <label for="inputPassword" translate>Password</label>
+                                            <input type="password" ng-model="password" class="form-control input-lg" id="inputPassword">
+                                          </div>
+                                          <div class="form-group">
+                                              <button type="submit" ng-click="login(username, password)" class="btn btn-primary" translate>Login</button>
+                                              <!-- TODO: if anonymous user is activate -->
+                                              <button type="submit" class="btn btn-default" translate>Continue as guest</button>
+                                          </div>
+                                        </form>
+                                    </div>
+                                </div>
+                            </div>
                         </div>
                     </div>
                 </div>
+                <!-- language switcher -->
+                <div class="btn-group" ng-controller="LanguageCtrl">
+                    <a class="btn btn-default">
+                        <i class="fa fa-flag"></i></a>
+                    <a class="btn btn-default dropdown-toggle" data-toggle="dropdown">
+                        <span class="fa fa-caret-down"></span></a>
+                    <ul class="dropdown-menu dropdown-menu-right">
```

Comment:
Is it possible to make a more generic version of this dropdown menu, so new languages can easily be added? If you do not know a way right away, you can add an issue.